app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+class TasksController < ApplicationController
+  def index
+    @tasks = Task.all
+  end
+
+  def new
+    @task = Task.new
+  end
+
+  def create
+    @task = Task.create task_params
```

Comment:
I'm getting an error that `ERROR:  null value in column "completed_at" violates not-null constraint`, this means you have a field `completed_at` that can't be nil, but you're not setting it in your strong params.